src/Microsoft.Azure.WebJobs.Host/InvokeFunctionFilterAttribute.cs

```
@@ -0,0 +1,96 @@
+        {
+            ExecutingFilter = executingFilter;
+            ExecutedFilter = executedFilter;
+        }
+
+        /// <summary>
+        /// Executing filter
+        /// </summary>
+        public string ExecutingFilter { get; }
+
+        /// <summary>
+        /// Executed filter
+        /// </summary>
+        public string ExecutedFilter { get; }
+
+        /// <summary>
+        /// Call the requested function
+        /// </summary>
+        /// <returns></returns>
+        public override async Task OnExecutingAsync(FunctionExecutingContext executingContext, CancellationToken cancellationToken)
+        {
+            if (executingContext == null)
+            {
+                throw new ArgumentNullException("executingContext");
+            }
+            
+            // TODO: Implement invoke function logic
+            if (!string.IsNullOrEmpty(ExecutingFilter))
+            {
+                IReadOnlyDictionary<string, object> paramaters = executingContext.Arguments;
+                JobHost host = executingContext.Host;
+                MethodInfo methodInfo = null;
+
+                foreach (var type in host.Configuration.TypeLocator.GetTypes())
+                {
+                    methodInfo = type.GetMethods().SingleOrDefault(p => string.Compare(p.Name, ExecutingFilter, StringComparison.OrdinalIgnoreCase) == 0);
+                }
+
+                if (methodInfo != null)
+                {
+                    executingContext.Logger.LogInformation("Executing function from filter...");
+
+                    var parameterName = methodInfo.GetParameters()[0].Name;
+                    Dictionary<string, object> parameters = new Dictionary<string, object>();
+                    parameters.Add(parameterName, executingContext);
+
+                    try
+                    {
+                        await host.CallAsync(methodInfo, parameters, cancellationToken);
+                    }
+                    catch (Exception e)
```

>Exception [](start = 27, length = 9)

Looks like we swallow and log the exception. Should an exception here cause an abort?